### DISCUSSION TITLE
🚀 FR system offline message using a regular template and/or file.

### DISCUSSION THREAD

**Discussion by robinsowell** [ExpressionEngine/ExpressionEngine#3851] [Feature Requests]
### Suggest an idea for ExpressionEngine

User is trying to standardize workflow and have all templates saved/edited as files.  You can now do this with the system messages templates via https://docs.expressionengine.com/latest/control-panel/template-manager.html#custom-system-messages but I can't get the 'offline' message working that way.  In the code, it looks like it just flat pulls it from the database.

It would be nice to have it work the same as the other system messages or at least allow saving as a file.

**Comment by intoeetive** [comment]
We can't use EE templates for offline message, because it is designed to be shown during maitenance mode, when there's no guarantee EE tags will be parsed at all

Having it saved as file would  be a nice feature though
